api/db/affiliations.js

```diff
@@ -211,6 +230,14 @@ const updateAuthAffiliation = async ({
           .where({ id: newRoleId })
           .first();
 
+  // Check user is assigning a valid role
+  if (
+    !validAssignableRoles[`${changedByRoleName}`].includes(roleName) &&
+    roleName !== null
+  ) {
+    throw new Error('User is attempting to assign an invalid role');
+  }
+
```

Comment:
I was looking at the whole file just so I could see the flow, and I saw that State Staff and State contractors are still getting expiration dates. Only state admins should get expirations dates now, right @jeromeleecms?

Reply:
I thought State Staff + Contractors still expired. We set their expiration 1 Year from the date they are approved. I could be wrong though or it changed.